Source/Urho3D/Core/Variant.h

```diff
@@ -207,20 +206,35 @@ class CustomVariantValue
 
 private:
     /// Type info.
-    const std::type_info& typeInfo_;
+    const std::type_info* typeInfo_{};
```

Comment:
Any particular reason why this cant remain a const reference? It would still be sizeof(void*).

Reply:
Done